code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -1,4 +1,4 @@
-
+GLOBAL_LIST_INIT(fire_reactions, list(/datum/gas_reaction/plasmafire, /datum/gas_reaction/tritfire, /datum/gas_reaction/h2fire))
```

Comment:
Make it a reaction flag that builds this list/a list on ssair

Reply:
Might be a good idea to do the same thing for flammable gases too in another pr.